include/llvm/InitializePasses.h

```diff
@@ -336,6 +336,69 @@ void initializeVirtRegMapPass(PassRegistry&);
 void initializeVirtRegRewriterPass(PassRegistry&);
 void initializeWholeProgramDevirtPass(PassRegistry &);
 void initializeWinEHPreparePass(PassRegistry&);
+
+// @LOCALMOD-BEGIN
+void initializeAddPNaClExternalDeclsPass(PassRegistry&);
```

Comment:
Yes, the only place those initialize* methods are called is in the asmjs backend. They can't affect normal builds.